.github/ISSUE_TEMPLATE/bug-openapi-react-query.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "Bug report: openapi-fetch"
+about: For the openapi-react-query library
+title: ""
+labels: openapi-react-query, bug
+assignees: ""
+body:
+  - type: input
+    attributes:
+      label: Version
+      placeholder: x.x.x
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the bug. Provide either a screenshot or the full error message!
+    validations:
+      required: true
+  - type: textarea
+    id: repro
+    attributes:
+      label: Reproduction
+      description: How can this be reproduced / when did the error occur? Does the issue occur in a specific browser, or all browsers? Bonus points for a GitHub repository link.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected result
+      description: (In case it’s not obvious)
+    validations:
+      required: true
+  - type: checkboxes
+    id: checklist
+    attributes:
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://example.com).
+      options:
+        - label: I agree to follow this project's [Code of Conduct](https://github.com/openapi-ts/openapi-typescript/blob/main/CODE_OF_CONDUCT.md)
+          required: true
```

Comment:
This feels like unnecessary toil to me, especially since Github has quite good reminders already. For example, I get:

![image](https://github.com/user-attachments/assets/5bc4649f-b4a1-428b-9b4f-0a2e942d15c6)

Reply:
Yeah that’s true. We’ve been lucky to not have badly-behave people. This was just copied from another project. I’ll remove for now, and just add back if needed